components/LeagueEntries/LeagueEntries.interface.ts

```diff
@@ -6,4 +6,5 @@ export interface ILeagueEntriesProps {
   linkUrl: string;
   isEliminated?: boolean;
   isPickSet?: boolean;
-}
\ No newline at end of file
+  teamLogo: string;
```

Comment:
Actually, looking at this more, this should be optional. When a user hasn't selected a team yet, there won't be a team logo.

Reply:
I did some testing with this: required vs optional.

Even if we make it required, no teamLogo pops up so I'd like to leave as is. 

<img width="1120" alt="image" src="https://github.com/user-attachments/assets/2e7a3af0-af66-42b7-9720-118f37f4d1d6">

Reply:
Make it optional. When implementing this component, the IDE will force the user to put something in for teamLogo prop. Which, it is not needed.

Reply:
Ah, I see. Thanks for letting me know and I reverted it back to optional.